app-nodejs-html5/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "app-nodejs-html5",
+  "version": "0.0.1",
+  "description": "HTML5 Service Invocation Application",
+  "main": "app.js",
+  "scripts": {
```

Maybe this would be nice?
```suggestion
  "scripts": {
    "start": "node app.js",
```